Resources/doc/reference/custom_fragment.rst

```diff
@@ -0,0 +1,90 @@
+Create a Custom Fragment
+========================
+
+
+Define a fragment Service
+-------------------------
+
+First you need to create a class that extends ``Sonata\ArticleBundle\FragmentService\AbstractFragmentService``
+
+
+.. code-block:: php
+
+namespace Acme\DummyBundle\FragmentService;
```

Comment:
Not sure you fixed [this](https://travis-ci.org/sonata-project/SonataArticleBundle/jobs/149016747#L292). Shouldn't the block from 13 to 45 be indented?